packages/core/src/utils/styleUtils/ssrStyles.ts

```diff
@@ -143,22 +117,16 @@ export const detachExperienceStyles = (experience: Experience): string | undefin
           continue;
         }
 
-        const defaultPatternDivStyles: Record<string, string> = Object.fromEntries(
-          Object.entries(buildCfStyles({}))
-            .filter(([, value]) => value !== undefined)
-            .map(([key, value]) => [toCSSAttribute(key), value]),
-        );
-
-        // I create a hash of the object above because that would ensure hash stability
-        const styleHash = md5(JSON.stringify(defaultPatternDivStyles));
+        // TODO: Why are we creating default styles here and where are they rendered in the end?
```

Comment:
Hey Thomas, I think this was added back then for fixing a bug related to pattern not rendering correctly in SSR. So adding default styles to the div

```
<div data-test-id="assembly">
```
fixed the issue. 
However, I tested it by removing this and its rendering just fine in SSR. So IMO we can safely remove this. It will also reduce the complexity.